src/load.ts

```diff
@@ -88,10 +88,13 @@ export default async function load(
       contentType = mime.getType(schemaID) || "";
     } else {
       // load remote
-      const headers = new Headers();
```

Comment:
Minor change: ditching a library-generated `Headers()` class not shipped by core Node just makes TypeScript a bit happier. Replaced with just a standard object which works just as well.